modules/nf-core/parabricks/haplotypecaller/main.nf

```diff
@@ -0,0 +1,60 @@
+process PARABRICKS_HAPLOTYPECALLER {
+    tag "$meta.id"
+    label 'process_high'
+
+    // Exit if running this module with -profile conda / -profile mamba
+    if (workflow.profile.tokenize(',').intersect(['conda', 'mamba']).size() >= 1) {
+        exit 1, "Parabricks module does not support Conda. Please use Docker / Singularity / Podman instead."
+    }
+
+    container "nvcr.io/nvidia/clara/clara-parabricks:4.0.1-1"
+
+    input:
+    tuple val(meta), path(input), path(input_index), path(interval_file)
+    path fasta
+
+    output:
+    tuple val(meta), path("*.vcf"), emit: vcf
+    path "versions.yml",            emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def output_file = args =~ "gvcf" ? "${prefix}.g.vcf" : "${prefix}.vcf"
+    def interval_file_command = interval_file ? interval_file.collect{"--interval-file $it"}.join(' ') : ""
+    def copy_index_command = input_index ? "cp -L $input_index `readlink -f $input`.bai" : ""
```

Comment:
can this tool be used on cram files?

Reply:
I am a little wary of doing any file copying, normally I would expect nextflow to handle it. Is the file not staged properly?

Reply:
Hello Friederike. I've fixed all the other comments you've left here so far, I'm just waiting to get push access to Ben's repo so I can push the changes. 

But for this comment, Parabricks needs the bam index file to be a regular file in the same directory as the .bam file in order to run. I'm not sure how nextflow handles the files behind the scenes, but I'm guessing that this is here as a check because Parabricks wasn't always finding the bam and its index together. Is there a more elegant way we can guarantee this behavior?

Reply:
If you look here, there's more context about why there needs to be file copying or hard linking: https://github.com/nf-core/modules/pull/2555#pullrequestreview-1196148772

Reply:
Thanks @adamrtalbot that's a much better solution, I'll do that here!